### DISCUSSION TITLE
Certified 1500hz Level 11 on osu! lazer latency certifier (with video proof)

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#26301] [Q&A] [answered]
Support requested that I make a discussion post on GitHub for this.

I am giving video proof of certifying level 11 (1500hz) on the osu! lazer latency certifier. It was requested a while back that if anyone gets over level 7 (480hz) then they should reach out to help "better understand extreme needs if they exist". Stated here: [Lazer 2022.612.0 · changelog | osu! (ppy.sh)](https://osu.ppy.sh/home/changelog/lazer/2022.612.0). This actually recommends that I use an fps limit of 2000hz, which is above anything which I can set in-game (which I find funny).

I put the visual spacing scale down to 0 and maxed out the approach rate and bpm (to 12 and 300 respectively). The way which I am able to tell the difference between the two is by analysing which one is choppier / has more random or spaced stuttering around the middle. By stuttering, I mean breaks in the approach circles. I am able to get up to 1500hz if I use 8x refresh rate (which is 1920hz as I use a 240hz monitor). Otherwise, I am able to get up to level 8 with 1000hz. 

One thing which I don't understand is why there is not an option to use unlimited fps in the certifier, wouldn't that be a better way to compare if people can tell a difference above 1000hz? Maybe I'm confusing something, but I find that the fps limit is the limiting factor (no pun intended) in being able to differentiate between the two options.

Here is video proof, as well as some screenshots of results:

https://drive.google.com/file/d/14W7Kvwapo-clqhpfFnCRqpZBRrDepVdt/view?usp=sharing

1920hz best:
![image](https://github.com/ppy/osu/assets/129146357/705790aa-3034-4a0e-8658-3834fd3c890d)
![image](https://github.com/ppy/osu/assets/129146357/009da06c-6a7a-4c0b-bd17-e17d8bae8679)

1000hz best:
![image](https://github.com/ppy/osu/assets/129146357/2692a169-6b38-4f8d-b00d-aca3451604b3)
![image](https://github.com/ppy/osu/assets/129146357/3628a3bf-6b00-452c-854e-e63d59e92c59)

**Comment by Flamiii** [comment, marked as answer]
I think I had a similar situation when the latency certifier initially released (https://github.com/ppy/osu/discussions/19387). The latency certifier is more about being able to see if you can distinguish between the two in a reasonable timeframe similar to normal gameplay as opposed to micro-analyzing the animations of the approach circles. And while the sliders do allow you to turn up the settings to AR 12 and 300 BPM, I wouldn't say that really reflects normal gameplay for most players. If you need that to tell the difference, it's unlikely you would actually be able to perceive the difference during normal gameplay.

**Comment by peppy** [comment]
Being able to see frames due to lack of vsync doesn't really mean much at all, as has kind of been explained but not really.
